polling_stations/templates/fragments/polling_station_unknown.html

```diff
@@ -29,4 +29,10 @@ <h2>{% blocktrans %}Contact {{ council }}{% endblocktrans %}</h2>
     {% endif %}
 </p>
 
-<p><strong>{% blocktrans %}Polling stations are open from 7am to 10pm.{% endblocktrans %}</strong></p>
+<p><strong>
+    {% if has_city_of_london_ballots %}
+        {% blocktrans %}Polling stations are open from 8am to 8pm.{% endblocktrans %}
+    {% else %}
+        {% blocktrans %}Polling stations are open from 7am to 10pm.{% endblocktrans %}
+    {% endif %}
```

Comment:
This approach to polling station opening hours may end up being wrong in situations where there are multiple election types upcoming. For example, a City of London by-election is scheduled a week before GLA elections or something.

I think we really need to move to WDIV being more specific about what election(s) we're talking about to properly fix this, which is a bigger job.

Reply:
I agree this is isn't the place to fix this, but we do get a lot of people looking at our site using City of London postcodes even though they don't live there. Do you think it's worth changing the text to make it clear that the 8 to 8 rule is CoL specific?

Something like:

> Polling stations for City of London elections are oprn from 8am to 8pm. Elections to other organisations (eg GLA or Parliament) polling stations will be open from 7am to 10pm.

Reply:
I've added some extra clarifying text based on this suggestion in https://github.com/DemocracyClub/UK-Polling-Stations/pull/8147/commits/02e2ccee4b7a0fbd3e111efd9d461ec604ddd15a